app/views/map.template.njk

```diff
@@ -41,5 +70,26 @@
   const center = {{ center | dump | safe }}
   const sbi = {{ sbi }}
   map.displayMap(sbi, parcels, center)
+
+  const searchParcels = () => {
+    var input, filter, table, tr, td, i, txtValue;
+    input = document.getElementById("search");
+    filter = input.value.toLowerCase();
+    table = document.getElementById("parcels");
+    tr = table.getElementsByTagName("tr");
+
+    for (i = 0; i < tr.length; i++) {
+      th = tr[i].getElementsByTagName("th")[0];
+      if (th) {
+        txtValue = th.textContent || th.innerText;
+        if (txtValue.toLowerCase().indexOf(filter) > -1) {
+          tr[i].style.display = "";
+        } else {
+          tr[i].style.display = "none";
+        }
+      }
+    }
+  }
```

Comment:
There's a lot of breaches of StandardJs styling in this function.

- don't use `var` for declaring variables, ideally use `const` but if the value must change then use `let` instead. 
- don't use semi-colons
- quotes should be single not double
- variables should be declared individually, so instead of:
```
const input, filter, table
```
you would have
```
const input
const filter
const table
```
- `td` is declared but never used
- this should be in a file in the `frontend/js` folder and then minified with webpack then loaded on the map page.